src/callbacks_stage/positivity_zhang_shu_dg1d.jl

```diff
@@ -5,6 +5,17 @@
 @muladd begin
 #! format: noindent
 
+@inline function compute_u_mean(u::AbstractArray{<:Any, 3}, element,
+                                mesh::AbstractMesh{1}, equations, weights, dg::DGSEM)
+    u_mean = zero(get_node_vars(u, equations, dg, 1, element))
+    for i in eachnode(dg)
+        u_node = get_node_vars(u, equations, dg, i, element)
+        u_mean += u_node * weights[i]
+    end
+    # note that the reference element is [-1,1]^ndims(dg), thus the weights sum to 2
```

Comment:
```suggestion
    # normalize with the total volume
    # note that the reference element is [-1,1], thus the weights sum to 2
```
I'm not sure about leaving out the `ndims(dg)`. If not, I would probably still use `ndims(mesh)`.
But at least the first addition seems useful.